backend/api/v1/endpoints/chatbot/huggingface.py

```
@@ -20,64 +23,41 @@
+
 def extract_text_from_pdf(file: UploadFile):
     doc = fitz.open(stream=file.file.read(), filetype="pdf")
     text = ""
     for page in doc:
         text += page.get_text()
     return text
 
-# Step 2: Create a Conversational QA Chain
-def create_qa_chain(text: str):
-    # Text splitting
+def create_retriever(text: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_retriever`

(ANN201)